ros_ws/src/simulation/vesc_sim/src/vesc_sim_driver.cpp

```diff
@@ -0,0 +1,91 @@
+#include "vesc_sim_driver.h"
+#include "car_config.h"
+
+VESCSimDriver::VESCSimDriver()
+{
+    this->m_servo_position_subscriber =
+        this->m_node_handle.subscribe<std_msgs::Float64>("/commands/servo/position", 1,
+                                                         &VESCSimDriver::servoPositionCallback, this);
+
+    this->m_motor_speed_subscriber =
+        this->m_node_handle.subscribe<std_msgs::Float64>("/commands/motor/speed", 1, &VESCSimDriver::motorSpeedCallback,
+                                                         this);
+
+    this->m_motor_break_subscriber =
+        this->m_node_handle.subscribe<std_msgs::Float64>("/commands/motor/break", 1, &VESCSimDriver::motorBreakCallback,
+                                                         this);
+
+    this->m_left_rear_wheel_velocity_publisher =
+        this->m_node_handle.advertise<std_msgs::Float64>("/racer/left_wheel_back_velocity_controller/command", 10);
+    this->m_right_rear_wheel_velocity_publisher =
+        this->m_node_handle.advertise<std_msgs::Float64>("/racer/right_wheel_back_velocity_controller/command", 10);
+    this->m_left_front_wheel_velocity_publisher =
+        this->m_node_handle.advertise<std_msgs::Float64>("/racer/left_wheel_front_velocity_controller/command", 10);
+    this->m_right_front_wheel_velocity_publisher =
+        this->m_node_handle.advertise<std_msgs::Float64>("/racer/right_wheel_front_velocity_controller/command", 10);
+
+    this->m_left_steering_position_publisher =
+        this->m_node_handle.advertise<std_msgs::Float64>("/racer/left_steering_hinge_position_controller/command", 10);
+    this->m_right_steering_position_publisher =
+        this->m_node_handle.advertise<std_msgs::Float64>("/racer/right_steering_hinge_position_controller/command", 10);
```

Comment:
In our existing code we use constants for topic names. I'm not sure if it is better to use constants here, but it could help with the long lines.